lib/api/pokedex_api_spec/handler/pokemon_handler.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:pokedex/api/pokedex_api_spec/handler/api_client.dart';
+import 'package:pokedex/api/pokedex_api_spec/model/pokemon_model.dart';
+import 'package:pokedex/utils/constant.dart' as k;
+
+class PokemonApi {
+  final ApiClient apiClient;
+
+  PokemonApi(this.apiClient);
+
+  static const int limitAmount = 20;
+
+  Future<PokemonModel> getPokemonList() async {
+    final queryParams = <String, dynamic>{};
+
+    final baseUri = Uri.parse(apiClient.baseUrl);
+    final uri = baseUri.replace(queryParameters: queryParams, path: '${baseUri.path}${k.apiLimit}$limitAmount');
```

Comment:
@yohancai mb I think its better to not put the limit param in the constants and adjust the base Uri since "/pokemon" is a different path. Also make the limitAmount private.